documents/decorators.py

```diff
@@ -0,0 +1,19 @@
+from flask import jsonify, request
```

Comment:
why is this file nested under documents? 
I'd assume these decorators would be used by all modules no?